pkg/lib/indexer/indexer.go

```diff
@@ -696,3 +800,152 @@ func (i ImageIndexer) DeprecateFromIndex(request DeprecateFromIndexRequest) erro
 
 	return nil
 }
+
+func ensureCSVFields(bundleDir string, bundleInfo bundleExportInfo) error {
+	dirContent, err := ioutil.ReadDir(bundleDir)
+	if err != nil {
+		return fmt.Errorf("error reading bundle directory %s, %v", bundleDir, err)
+	}
+
+	for _, f := range dirContent {
+		if f.IsDir() {
+			continue
+		}
+		unstructuredCSV := unstructured.Unstructured{}
+
+		csvFile, err := os.OpenFile(path.Join(bundleDir, f.Name()), os.O_RDONLY, 0)
+		if err != nil {
+			continue
+		}
+		err = k8syaml.NewYAMLOrJSONDecoder(csvFile, 30).Decode(&unstructuredCSV)
+		csvFile.Close()
+		if err != nil {
+			continue
+		}
+
+		if unstructuredCSV.GetKind() != operators.ClusterServiceVersionKind {
+			continue
+		}
+
+		csv := pregistry.ClusterServiceVersion{}
+		if err := runtime.DefaultUnstructuredConverter.FromUnstructured(unstructuredCSV.UnstructuredContent(),
+			&csv); err != nil {
+			return err
+		}
+		csvReplaces, err := csv.GetReplaces()
+		if err != nil {
+			return err
+		}
+		csvSkips, err := csv.GetSkips()
+		if err != nil {
+			return err
+		}
+		sort.Strings(csvSkips)
```

Comment:
Why is this sort necessary? Shouldn't we be able to do this in any arbitrary order?